src/styles/sponsors.module.css

```diff
@@ -0,0 +1,120 @@
+.jumbotron {
+    background-image: url(https://images.unsplash.com/photo-1519999482648-25049ddd37b1?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=0471f46a1cda51342fdecf9a8c42349f&auto=format&fit=crop&w=1852&q=80);
+    background-size: cover;
+}
+
+.heading {
+    font-size: 6em;
+    font-weight: 3em;
+    color: #ffffff;
+}
+
+.headingContainer {
+    margin-bottom: 2em;
+    border-bottom: 3px solid #eeeeee;
+}
+
+.blurb {
+    font-size: 2.5em;
+    font-weight: 2em;
+}
+
+.sponsorsLevel {
```

Comment:
do you think we can use bold and stuff from bootstrap?

https://getbootstrap.com/docs/4.1/utilities/text/#font-weight-and-italics

```<p class="lead font-weight-bold text-uppercase">Bold text.</p>````

Reply:
I think full uppercase doesn't look the greatest... Changed to `lead font-weight-bold` though.